tests/common/Smi.Common.Tests/NuspecIsCorrectTests.cs

```diff
@@ -118,10 +118,6 @@ public void TestDependencyCorrect(string csproj, string nuspec, string packagesM
                         if (Regex.IsMatch(line, @"[\s[]" + Regex.Escape(package) + @"[\s\]]", RegexOptions.IgnoreCase))
                         {
                             int count = new Regex(Regex.Escape(version)).Matches(line).Count;
```

Comment:
This line can probably be removed as well?

Reply:
Good spot